portugues/colabora/fork/comdiferencas/diff/pull/pull.md

```diff
@@ -7,3 +7,6 @@ O pull request não foi aceito pois havia um erro ortográfico em um comentário
 [Corrijo os erros ortográficos com a ajuda de um amigo que manja de inglês e reenvio.](reenvio/reenviar.md)
 
 [Descobre, logo agora que aprendeu o bash, vai ter que retomar os estudos de inglês no duolingo] (descobre/descobrir.md)
+
+Abro o [Google Tradutor](https://translate.google.com.br/) e verifico a tradução do meu comentário. 
+Não é o que eu queria escrever: [corrijo meu erro e reenvio](reenvio/reenviar.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.